src/utils/HgvValidations.ts

```diff
@@ -0,0 +1,308 @@
+  "3",
+  "v",
+  "4",
+  "7",
+  "5"
+];
+
+const vehicleClassDescription: string[] = [
+  "motorbikes over 200cc or with a sidecar",
+  "not applicable",
+  "small psv (ie: less than or equal to 22 seats)",
+  "motorbikes up to 200cc",
+  "trailer",
+  "large psv(ie: greater than 23 seats)",
+  "3 wheelers",
+  "heavy goods vehicle",
+  "MOT class 4",
+  "MOT class 7",
+  "MOT class 5"
+];
+
+const vehicleConfiguration: string[] = [
+  "rigid",
+  "articulated",
+  "centre axle drawbar",
+  "semi-car transporter",
+  "semi-trailer",
+  "low loader",
+  "other",
+  "drawbar",
+  "four-in-line",
+  "dolly",
+  "full drawbar"
+];
+
+const euVehicleCategory: string[] = [
+  "m1",
+  "m2",
+  "m3",
+  "n1",
+  "n2",
+  "n3",
+  "o1",
+  "o2",
+  "o3",
+  "o4"
+];
+
+const approvalType: string[] = [
+  "NTA",
+  "ECTA",
+  "IVA",
+  "NSSTA",
+  "ECSSTA"
+];
+
+const bodyTypeDescription: string[] = [
+  "articulated",
+  "single decker",
+  "double decker",
+  "other",
+  "petrol/oil tanker",
+  "skeletal",
+  "tipper",
+  "box",
+  "flat",
+  "refuse",
+  "skip loader",
+  "refrigerated"
+];
+
+const microfilmDocumentType: string[] = [
+  "PSV Miscellaneous",
+  "AAT - Trailer Annual Test",
+  "AIV - HGV International App",
+  "COIF Modification",
+  "Trailer COC + Int Plate",
+  "RCT - Trailer Test Cert paid",
+  "HGV COC + Int Plate",
+  "PSV Carry/Auth",
+  "OMO Report",
+  "AIT - Trailer International App",
+  "IPV - HGV EEC Plate/Cert",
+  "XCV - HGV Test Cert free",
+  "AAV - HGV Annual Test",
+  "COIF Master",
+  "Tempo 100 Sp Ord",
+  "Deleted",
+  "PSV N/ALT",
+  "XPT - Tr Plating Cert paid",
+  "FFV - HGV First Test",
+  "Repl Vitesse 100",
+  "TCV - HGV Test Cert",
+  "ZZZ -  Miscellaneous",
+  "Test Certificate",
+  "XCT - Trailer Test Cert free",
+  "C52 - COC and VTG52A",
+  "Tempo 100 Report",
+  "Main File Amendment",
+  "PSV Doc",
+  "PSV COC",
+  "PSV Repl COC",
+  "TAV - COC",
+  "NPT - Trailer Alteration",
+  "OMO Certificate",
+  "PSV Repl COIF",
+  "PSV Repl COF",
+  "COIF Application",
+  "XPV - HGV Plating Cert Free",
+  "TCT  - Trailer Test Cert",
+  "Tempo 100 App",
+  "PSV Decision on N/ALT",
+  "Special Order PSV",
+  "NPV - HGV Alteration",
+  "No Description Found",
+  "Vitesse 100 Sp Ord",
+  "Brake Test Details",
+  "COIF Productional",
+  "RDT - Test Disc Paid",
+  "RCV -  HGV Test Cert",
+  "FFT -  Trailer First Test",
+  "IPT - Trailer EEC Plate/Cert",
+  "XDT - Test Disc Free",
+  "PRV - HGV Plating Cert paid",
+  "COF Cert",
+  "PRT - Tr Plating Cert paid",
+  "Tempo 100 Permit"
+];
+
+const plateReasonForIssue: string[] = [
+  "Free replacement",
+  "Replacement",
+  "Destroyed",
+  "Provisional",
+  "Original",
+  "Manual"
+];
+
+const fitmentCode: string[] = [
+  "double",
+  "single"
+];
+
+const populateVehicleClassCode = (parent: any, helpers: any) => {
+  // if (parent.description === "motorbikes over 200cc or with a sidecar") {
+  //   return "2";
+  // }
+  // throw new Error("Not valid");
+  return "2";
```

Comment:
yes, these will be implemented soon on 10213 and 10214 (which takes care of autopopulation). They are hardcoded for now because otherwise the validation will fail and I needed to populate somehow these fields.

Reply:
Okey Dokey